specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/schedule.json

```diff
@@ -247,9 +247,6 @@
           "200": {
             "description": "OK"
           },
-          "404": {
```

Comment:
I'm confused about why we are removing 404 here. If we try to get a schedule, shouldn't we return a 404 if it doesn't exist?

Reply:
In the specs, we only specify success cases and not the failure cases. If we start specifying error cases, the SDK does not throw an exception and treats it as a normal case.

For more info, @finiteattractor can describe it in more details.

Reply:
Thank you! Signing off.